docs/docs.go

```diff
@@ -0,0 +1,107 @@
+// GENERATED BY THE COMMAND ABOVE; DO NOT EDIT
+// This file was generated by swaggo/swag at
+// 2018-12-17 22:19:48.312668 +0100 CET m=+0.051467374
+
+package docs
+
+import (
+	"bytes"
+
+	"github.com/alecthomas/template"
+	"github.com/swaggo/swag"
```

Comment:
isn't go-swagger more appropriate? seems to be more popular

Reply:
I'll take a look on go-swagger as well, this is the first thing I found. Quick look suggests that they are very similar, but I'll check how do they differ.